### PR TITLE
Add Robotframework serialLibrary package

### PR DESCRIPTION
**Pull request by jdmarques**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [LICENSE](https://github.com/conda-forge/staged-recipes/pull/21337/files#diff-64a2fe248001eb0617c9459900b24ab3de0da0edaa09935cdd94d33bdfc48552)).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/robotframework-seriallibrary/LICENSE [hunk 1]
```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2016, Yasushi Masuda
```
Comment (bollwyvl) [inline recipes/robotframework-seriallibrary/LICENSE:1]: I'd recommend also making an upstream PR to include this in the sdist

**Comment by bollwyvl** [comment]
Opened https://github.com/whosaysni/robotframework-seriallibrary/pull/23 ... you may be able to grab some of those [patches](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#patches)... but maybe give it a few days to see if the upstream responds.

**Comment by jdmarques** [comment]
> Opened [whosaysni/robotframework-seriallibrary#23](https://github.com/whosaysni/robotframework-seriallibrary/pull/23) ... you may be able to grab some of those [patches](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#patches)... but maybe give it a few days to see if the upstream responds.

**Comment by jdmarques** [comment]
@bollwyvl thanks for the review and the help! I was most likely to go around in circles trying to figure out the issue. 
I Will keep an eye on the upstream PR

**Comment by bollwyvl** [comment]
I'd say at this point, you should probably go ahead and patch. Unfortunately, as the upstream PR is patching the _input_ to the build system, not the output, you can't use it directly.

I've made [this PR](https://github.com/jdmarques/staged-recipes/pull/1) which you can merge into yours which applies the patch and tidies up some things. I'm also happy to help maintain, and have added myself.

**Comment by bollwyvl** [comment]
@conda-forge-admin please rerender

**Comment by bollwyvl** [comment]
@conda-forge-admin please rerender

**Comment by jdmarques** [comment]
@conda-forge-admin, please ping team

**Comment by jdmarques** [comment]
conda-forge/help-python

**Comment by ocefpaf** [comment]
@conda-forge-admin, please restart ci

**Pull request merged**